### PR TITLE
6 BLB cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/for_the_common_good.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:For the Common Good
+ManaCost:X X G
+Types:Sorcery
+A:SP$ CopyPermanent | ValidTgts$ Permanent.token+YouCtrl | NumCopies$ X | AddTriggers$ TrigFight | SubAbility$ DBPumpAll | TgtPrompt$ Create X tokens that are copies of target token you control. Then tokens you control gain indestructible until your next turn. You gain 1 life for each token you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/for_the_common_good.txt:4]: wrong TgtPrompt

File: forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Harvestrite Host
+ManaCost:2 W
+Types:Creature Rabbit Citizen
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Rabbit.Other+YouCtrl | Execute$ TrigPump | TriggerDescription$ When CARDNAME or another Rabbit you control enters, target creature you control gets +1/+0 until end of turn. Then draw a card if this is the second time this ability has resolved this turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | NumAtt$ +1 | SubAbility$ DBDraw
+SVar:PlayMain1:TRUE
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ Resolved | ConditionSVarCompare$ EQ2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt:8]: SVar doesn't exist

File: forge-gui/res/cardsfolder/upcoming/wishing_well.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | Execute$ TrigPlay | ConditionDefined$ Remembered | ConditionPresent$ Card | TriggerDescription$ When you do, you may cast target instant or sorcery card with mana value equal to the number of coin counters on CARDNAME from your graveyard without paying its mana cost. If that spell would be put into your graveyard, exile it instead. Activate only as a sorcery.
+SVar:TrigPlay:DB$ Play | TgtZone$ Graveyard | ValidTgts$ Instant.YouCtrl+cmcEQX,Sorcery.YouCtrl+cmcEQX | ValidSA$ Spell | TgtPrompt$ Choose target instant or sorcery card with mana value equal to the number of coin counters on CARDNAME from your graveyard | WithoutManaCost$ True | Optional$ True | ReplaceGraveyard$ Exile | AILogic$ ReplaySpell | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/wishing_well.txt:6]: cleanup should be called by the trigger line instead

File: forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt (hunk 2 above)
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt:7]: I don't understand? It's referred to in TrigPump?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt:7]: the Condition one

File: forge-gui/res/cardsfolder/upcoming/for_the_common_good.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/for_the_common_good.txt]: now it exists twice

**Review by tool4ever — APPROVED** [review]

**Pull request merged**